### ISSUE TITLE
Not able to set hotkey for the qtranslate extension

### ISSUE THREAD

**Issue by rp1231** [poochie89/SnipDo-Extensions#110]
The hot key doesn't get saved for some reason.

**Comment by poochie89** [comment]
Do you mean you always get the prompt to set the key?

**Comment by rp1231** [comment]
No, What I mean is that I set the hot key via the prompt but it doesn't get set. When I reopen the prompt it just shows the A key and not the hot key that was saved.

**Comment by rp1231** [comment]
Ok, my bad.
It does get set. but when I click on the edit key, it shows the A key instead of the currently set hotkey.
That led me to think that it was not being set.
I think that it would be good from a user experience perspective to show the currently set hotkey when the `SET` button is pressed instead of the `A` key

**Comment by rp1231** [comment]
Also when the edit button is clicked,
It should show this if the hot key has already been set:
<img width="366" alt="SnipDo_UmJe8xHeEl" src="https://github.com/poochie89/SnipDo-Extensions/assets/47380919/303c7266-1066-48bf-a3a2-fcf7abde6668">
Insead of this:
<img width="366" alt="SnipDo_OqnGBOHLQo" src="https://github.com/poochie89/SnipDo-Extensions/assets/47380919/62b7012e-5e8a-4e70-9f76-bd3544950f8c">
